src/components/Dashboard/TodoList.vue

```diff
@@ -352,7 +352,7 @@
                 </template>
 
                 <!-- default task -->
-                <template v-if="item.isDefaultTask">
+                <template v-else-if="item.isDefaultTask">
```

Comment:
Looks right. This block (and all blocks below) need to be mutually exclusive, per line 343.

👍